docker/docker-compose.yml

```
@@ -0,0 +1,45 @@
+version: "3.8"
+
+services:
+  zebra:
+    image: zfnd/zebra
+    platform: linux/amd64
+    build:
+      context: ../
+      dockerfile: docker/Dockerfile
+      target: runtime
+    restart: unless-stopped
+    deploy:
+      resources:
+        reservations:
+          cpus: "4"
+          memory: 16G
+    env_file:
+      - .env
+    #! Uncomment the following line to use a zebrad.toml from the host machine
+    #! NOTE: This will override the zebrad.toml in the image and make some variables irrelevant
+    # configs:
+    #   - source: zebra_config
+    #     target: /etc/zebrad/zebrad.toml
+    #     uid: '2001' # Rust's container default user uid
+    #     gid: '2001' # Rust's container default group gid
+    #     mode: 0440
+    volumes:
+      - zebrad-cache:/var/cache/zebrad-cache
+    ports:
+      # Zebra uses the following inbound and outbound TCP ports
```

###### Optional Suggestion

```suggestion
      # Zebra uses the following default inbound and outbound TCP ports
```